docs/references/ic-interface-spec.md

```diff
@@ -1515,38 +1515,46 @@ defaulting to `I = i32` if the canister declares no memory.
+    ic0.stable64_read : (dst : i64, offset : i64, size : i64) -> ();                                  // * s
+
+    ic0.certified_data_set : (src : I, size : I) -> ();                                               // I G U Ry Rt T
+    ic0.data_certificate_present : () -> i32;                                                         // *
+    ic0.data_certificate_size : () -> I;                                                              // NRQ CQ
+    ic0.data_certificate_copy : (dst : I, offset : I, size : I) -> ();                                // NRQ CQ
+
+    ic0.time : () -> (timestamp : i64);                                                               // *
+    ic0.global_timer_set : (timestamp : i64) -> i64;                                                  // I G U Ry Rt C T
+    ic0.performance_counter : (counter_type : i32) -> (counter : i64);                                // * s
+    ic0.is_controller : (src : I, size : I) -> ( result : i32);                                       // * s
+    ic0.in_replicated_execution : () -> (result : i32);                                               // * s
     
-    ic0.cost_call : (method_name_size: i64, payload_size : i64, dst : I) -> ();           // * s
-    ic0.cost_create_canister : (dst : I) -> ();                                           // * s
-    ic0.cost_http_request : (request_size : i64, max_res_bytes : i64, dst : I) -> ();     // * s
-    ic0.cost_sign_with_ecdsa : (src : I, size : I, ecdsa_curve: i32, dst : I) -> i32;     // * s
-    ic0.cost_sign_with_schnorr : (src : I, size : I, algorithm: i32, dst : I) -> i32;     // * s
-    ic0.cost_vetkd_derive_key : (src : I, size : I, vetkd_curve: i32, dst : I) -> i32;  // * s
+    ic0.cost_call : (method_name_size: i64, payload_size : i64, dst : I) -> ();                       // * s
+    ic0.cost_create_canister : (dst : I) -> ();                                                       // * s
+    ic0.cost_http_request : (request_size : i64, max_res_bytes : i64, dst : I) -> ();                 // * s
+    ic0.cost_sign_with_ecdsa : (src : I, size : I, ecdsa_curve: i32, dst : I) -> i32;                 // * s
+    ic0.cost_sign_with_schnorr : (src : I, size : I, algorithm: i32, dst : I) -> i32;                 // * s
+    ic0.cost_vetkd_derive_key : (src : I, size : I, vetkd_curve: i32, dst : I) -> i32;                // * s
 
-    ic0.debug_print : (src : I, size : I) -> ();                                          // * s
-    ic0.trap : (src : I, size : I) -> ();                                                 // * s
+    ic0.env_var_count : () -> I;                                                                      // *
+
+    ic0.env_var_name_size : (index: I) -> I;                                                          // *
+    ic0.env_var_name_copy : (dst: I, offset: I, size: I, index: I) -> ();                             // *
```

Comment:
Maybe it'd make more sense to put `index` first: analogously to `name_src` and `name_size` being first for `ic0.env_var_value_{size,copy}`. @AlexandraZapuc